pkg/ingester/limiter.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 const (
-	errMaxSeriesPerMetricLimitExceeded   = "per-metric series limit (local limit: %d global limit: %d actual local limit: %d) exceeded"
-	errMaxSeriesPerUserLimitExceeded     = "per-user series limit (local limit: %d global limit: %d actual local limit: %d) exceeded"
-	errMaxMetadataPerMetricLimitExceeded = "per-metric metadata limit (local limit: %d global limit: %d actual local limit: %d) exceeded"
-	errMaxMetadataPerUserLimitExceeded   = "per-user metric metadata limit (local limit: %d global limit: %d actual local limit: %d) exceeded"
+	errMaxSeriesPerMetricLimitExceeded   = "per-metric series limit of %d exceeded. Contact administrator to raise it. (local limit: %d global limit: %d actual local limit: %d) exceeded"
```

Comment:
super-nit: error messages are typically single-sentence, without dot at the end. So perhaps: `per-metric series limit of %d exceeded, please contact administrator to raise it (local limit:...)`